pkg/controller/snatglobalinfo.go

```diff
@@ -231,6 +231,12 @@ func (cont *AciController) handleSnatNodeInfo(nodeinfo *nodeinfo.NodeInfo) bool
 		ret = cont.deleteNodeinfoFromGlInfoCache(nodename)
 		updated = true
 	} else {
+		// This case ignores any stale entry is present in nodeinfo
+		_, _, err := cont.nodeIndexer.GetByKey(nodename)
+		if err != nil {
+			cont.log.Error("Could not lookup node: ", err, "nodeName: ", nodename)
```

Comment:
hate to nit pick, but this should not be an "error" level, right? this should perhaps just be at "info" level?

Reply:
Yeah good point